app/assets/javascripts/dashboard/dashboard.js

```diff
@@ -744,3 +808,120 @@ $('body').on('click', '.role-btn-group .btn', function(e) {
   handleBootstrapErrors();
   handleUnassignedErrors();
 });
+
+$('body').on('click', '.remove-node-link', function (e) {
+  var $this = $(this);
+  var id = $this.data('id');
+
+  e.preventDefault();
+  removePendingAcceptance(id);
+
+  if ($this.hasClass('disabled')) {
+    return;
+  }
+
+  if (confirm('Are you sure?')) {
```

Comment:
I think it wouldn't harm to also include the minion hostname and/or id on the confirmation message.